### DISCUSSION TITLE
nuxt version 3

### DISCUSSION THREAD

**Discussion by mdshakil001** [nuxt/nuxt#16613] [General]
It seems that the documentation page is complete, so is nuxt.js stable now and  can i use it in production ?

**Comment by TheAlexLichter** [comment]
Many projects use Nuxt 3 already in production. The stable version is yet to be released. It will be a few more RCs, as written in #8067  ☺️ 

> Note
> This PR includes draft notes for the next release version of Nuxt 3. We plan to release 1-2 more release candidates before the final 3.0.0 version.
